packages/stack/src/views/Stack/Card.tsx

```diff
@@ -480,7 +480,10 @@ export default class Card extends React.Component<Props> {
       <CardAnimationContext.Provider value={animationContext}>
         <View pointerEvents="box-none" {...rest}>
           {overlayEnabled ? (
-            <View style={StyleSheet.absoluteFill}>
+            <View
+              pointerEvents="none"
```

Comment:
Thanks! Can you change it to `box-none` so the user can customize it by specifying it in the overlay itself?

```suggestion
              pointerEvents="box-none"
```

Reply:
Cool by me - done